_sass/_typography.scss

```diff
@@ -0,0 +1,27 @@
+article.post.typography{
+  .blender-samples{
+    .title{
+      @include span-columns(3);
+    }
+    .typo-example{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space